quinn-proto/src/connection/streams/state.rs

```diff
@@ -240,8 +287,12 @@ impl StreamsState {
 
         // Stopped streams become closed instantly on FIN, so check whether we need to clean up
         if closed {
-            self.recv.remove(&id);
-            self.stream_freed(id, StreamHalf::Recv);
+            let recv = self
```

Comment:
Maybe `let rs` for clarity that this is the same value, just owned rather than borrowed?

Reply:
done